### ISSUE TITLE
application fails to start with 2.1.0

### ISSUE THREAD

**Issue by jmls** [chiefy/vaulted#51]
Using vaulted@2.1.0 my app has this startup error

```
Error: Configuration property "ssl_ciphers" is not defined
```

the appropriate stack trace line is this

```
  at [object Object].Config.get (/workspace/borg/node_modules/vaulted/node_modules/config/lib/config.js:177:11)
  at getSSLOptions (/workspace/borg/node_modules/vaulted/lib/api.js:52:28)
  at load (/workspace/borg/node_modules/vaulted/lib/api.js:96:33)
  at loadAPIDefinitions (/workspace/borg/node_modules/vaulted/lib/api.js:127:10)
  at new API (/workspace/borg/node_modules/vaulted/lib/api.js:147:28)
  at new Vaulted (/workspace/borg/node_modules/vaulted/lib/vaulted.js:28:14)
```

reverting back to 2.0.0 removes the error

**Comment by kenjones-cisco** [comment]
The following files in the `config` directory are required.
Need to update `package.json` to include:

``` json
files: [
"config"
]
```

That should result in the default configuration files being copied and installed as well.

**Comment by kenjones-cisco** [comment]
Resolved with commit 3d20fb2a220ac7136dfac656238fd3855d547028
